README.md

```diff
@@ -11,89 +11,27 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
-
-Use the Helm chart to install the latest version of Kubeapps:
-
-```bash
-helm repo add bitnami https://charts.bitnami.com/bitnami
-helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
-```
-
-> **IMPORTANT** This assumes an insecure Helm installation, which is not recommended in production. See [the documentation to learn how to secure Helm and Kubeapps in production](docs/user/securing-kubeapps.md).
-
-The above commands will deploy Kubeapps into the `kubeapps` namespace in your cluster, it may take a few seconds to execute. For more information on installing and configuring Kubeapps, checkout the [chart README](chart/kubeapps/README.md).
-
-Once it has been deployed and the Kubeapps pods are running, port-forward to access the Dashboard:
-
-```bash
-export POD_NAME=$(kubectl get pods -n kubeapps -l "app=kubeapps,release=kubeapps" -o jsonpath="{.items[0].metadata.name}")
-echo "Visit http://127.0.0.1:8080 in your browser to access the Kubeapps Dashboard"
-kubectl port-forward -n kubeapps $POD_NAME 8080:8080
-```
-
-![Dashboard login page](docs/img/dashboard-login.png)
-
-Access to the dashboard requires a Kubernetes API token to authenticate with the Kubernetes API server. Read the [Access Control](docs/user/access-control.md) documentation for more information on configuring users for Kubeapps.
-
-The following commands create a ServiceAccount and ClusterRoleBinding named `kubeapps-operator` which will enable the dashboard to authenticate and manage resources on the Kubernetes cluster:
-
-```bash
-kubectl create serviceaccount kubeapps-operator
-kubectl create clusterrolebinding kubeapps-operator --clusterrole=cluster-admin --serviceaccount=default:kubeapps-operator
-```
-
-Use the following command to reveal the authorization token that should be used to authenticate the Kubeapps dashboard with the Kubernetes API:
-
-```bash
-kubectl get secret $(kubectl get serviceaccount kubeapps-operator -o jsonpath='{.secrets[].name}') -o jsonpath='{.data.token}' | base64 --decode
-```
-
-**NOTE**: It's not recommended to create cluster-admin users for Kubeapps. Please refer to the [Access Control](docs/user/access-control.md) documentation to configure more fine-grained access.
-
-![Dashboard main page](docs/img/dashboard-home.png)
-
-To remove Kubeapps from your cluster, simply run:
-
-```bash
-helm delete --purge kubeapps
-kubectl delete crd apprepositories.kubeapps.com
-```
-
-To delete the `kubeapps-operator` ServiceAccount and ClusterRoleBinding,
-
-```bash
-kubectl delete clusterrolebinding kubeapps-operator
-kubectl delete serviceaccount kubeapps-operator
-```
-
-## Build from Source
-
-Please refer to the [Kubeapps Build Guide](docs/developer/build.md) for instructions on setting up the build environment and building Kubeapps from source.
+For instructions of how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
```

Comment:
What about just having quick Helm chart instructions here and then link to getting started for the more detailed installation instructions? I think it's useful to have, even if it just helps point out that it's installable via a chart.

```
helm repo add bitnami https://charts.bitnami.com/bitnami
helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
```

```suggestion
For instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
```